### PR TITLE
Change Graph.InstallEsx to Graph.InstallESXi

### PR DESCRIPTION
**Pull request by anhou**
resolve RackHD/RackHD#218
taksgraph change https://github.com/RackHD/on-taskgraph/pull/139

@RackHD/corecommitters

### CODE REVIEW

File: lib/task-data/tasks/install-esx.js [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 module.exports = {
     friendlyName: 'Install Esx',
-    injectableName: 'Task.Os.Install.Esx',
+    injectableName: 'Task.Os.Install.ESXi',
```
Comment (yyscamper) [inline lib/task-data/tasks/install-esx.js]: I think other places need also be changed:
-  The filename.
- the kickstart/boot-config file name.
- the osType.
- the properties.os.hypervisor.

Comment (anhou) [inline lib/task-data/tasks/install-esx.js]: they're not intended to be included in this PR, these PRs are only for user's interfaces,  not for code details. suggest to refine them during developing bit by bit.

**Comment by iceiilin** [comment]
:+1:

File: lib/task-data/tasks/install-esx.js (hunk 1 above)
Comment (yyscamper) [inline lib/task-data/tasks/install-esx.js]: OK, make sense.

**Comment by yyscamper** [comment]
:+1:

**Pull request merged**